tests/resources/integration_test_data/assetized_child_ref_non_assetized_grandchild/floor1.usd

```diff
@@ -0,0 +1,16 @@
+#usda 1.0
+def "ParkingLot_Floor"
+{
+    def "Car1" (
+        references = @car.usd@</Car>
+    )
+    {
+        color3f color = (0, 0, 1)
```

Comment:
[consider] We could simplify and remove these overrides, since it doesn't look like we need these for testing. I found myself scanning the code for asserts on the correct value of `color`, which is never done, so is a bit misleading.

We could use the `color` override to assert that the expected child was chosen (e.g. to assert that local relative has precedence over search-path relative), but I think that would be testing `ArDefaultResolver` and not affected by our integration(?).

Reply:
I think the point is more general. It would be good to simplify the dataset to the absolute bare minimum required for the test(s).

Reply:
I think we can cul us down to only having a single car on the floor, now that we're not accessing the cars in different ways, (the multiple cars was to test caching initially anyhow, so you're right they're not appropriate,) there's also no need for the duplicate and override properties yeah.

Reply:
Id like to keep the two cars for the reasons elliot described, but agree that colors are tangential at this point.

Reply:
https://github.com/OpenAssetIO/usdOpenAssetIOResolver/pull/13/commits/15b8c3684dffefe00a8d83aa05f309407a86c82c

Removed redundant properties.